### ISSUE TITLE
distal radioulnar joint (DRUJ) space widening

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10746]
**Preferred term label:**
distal radioulnar joint (DRUJ) space widening

**Synonyms**

**Definition (free text, please give PubMed ID)**
Increased space between the distal radius and distal ulna at sigmoid notch; radioulnar distance >6mm at the sigmoid notch

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal radioulnar joint (DRUJ) instability
Distal radius fracture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:5000040
- label: Distal radioulnar joint space widening
- parents: [Abnormality of the radioulnar joints (HP:0003059)](https://hpo.jax.org/app/browse/term/HP:0003059)
- pmid: 
- def: Increased space between the distal radius and distal ulna at sigmoid notch (radioulnar distance more than 6mm at the sigmoid notch).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10746
- synonyms:
